src/main/java/ch/cern/eam/wshub/core/services/workorders/impl/ChecklistServiceImpl.java

```diff
@@ -114,14 +104,13 @@ private void getResponsibilityDescriptions(InforContext context, WorkOrderActivi
 						.forEach(signature -> signature.setResponsibilityDescription(getCellContent.apply(row, "description"))));
 	}
 
-	private WorkOrderActivityChecklistSignatureResult[] filterSignatures(WorkOrderActivityCheckListDefaultResult workOrderActivityCheckList) throws Exception {
-		DateAdapter dateAdapter = new DateAdapter();
+	private WorkOrderActivityChecklistSignatureResult[] filterSignatures(WorkOrderActivityCheckListDefaultResult workOrderActivityCheckList){
 		String reviewerQualification = workOrderActivityCheckList.getReviewerQualification();
 		String performer1Qualification = workOrderActivityCheckList.getPerformer1Qualification();
 		String performer2Qualification = workOrderActivityCheckList.getPerformer2Qualification();
 		List<String> qualifications = new ArrayList<>();
-		for(USERDEFINEDCODEID_Type userData : workOrderActivityCheckList.getUserResponsibilities())
-			qualifications.add(userData.getUSERDEFINEDCODE());
+		for(UserQualification qualification : workOrderActivityCheckList.getUserQualifications())
+			qualifications.add(qualification.getUserDefinedCode());
```

Comment:
Can't a map be used here with the `getUserDefinedCode` function to convert `UserQualification[]` to `List<String>`?

Reply:
Fixed.